build.rs

```diff
@@ -0,0 +1,13 @@
+use cc::Build;
+use std::env;
+
+fn main() {
+    let target = env::var("TARGET").unwrap();
+    // Right now, JIT only supports linux and Mac OS on x86_64 CPUs
+    if target.contains("x86_64") && (target.contains("linux") || target.contains("apple")) {
```

Comment:
`apple` is `vender` type, `linux` is  `sys` type.

Here use `darwin` would be better?